### DISCUSSION TITLE
Run Docker cp on a running Docker instance

### DISCUSSION THREAD

**Discussion by thiagoolsilva** [batect/batect#1162] [Q&A]
I came Here across to check if it is possible to run something like Docker cp to copy host files to a running container using batect.

**Comment by charleskorn** [comment]
Hi @thiagoolsilva, you can definitely use `docker cp` to copy host files into a running container started by Batect. However, you might find it more convenient to [mount a file or directory into the container](https://batect.dev/docs/reference/cheat-sheet/#mount-a-file-or-directory-into-a-container).

**Reply by thiagoolsilva** [reply]
Thanks for your reply @charleskorn . What is your suggestion to run the command Docker cp after batect started the container?

**Comment by thiagoolsilva** [comment]
I've tried to mount a volume into batect container on a Jenkins job, but it does not mount properly any file or directory on it. 
 
> Hi @thiagoolsilva, you can definitely use `docker cp` to copy host files into a running container started by Batect. However, you might find it more convenient to [mount a file or directory into the container](https://batect.dev/docs/reference/cheat-sheet/#mount-a-file-or-directory-into-a-container).

**Reply by charleskorn** [reply]
Is your Jenkins job running inside a container, and using the Docker daemon outside the container? If so, that might be the cause of this issue.
